sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Database.java

```diff
@@ -16,6 +18,16 @@
  */
 public final class Database extends Resource {
 
+    /**
+     * Constructor.
+     *
+     * @param objectNode the {@link ObjectNode} that represent the
+     * {@link JsonSerializable}
+     */
+    public Database(ObjectNode objectNode) {
```

Comment:
Does this needs to be public ?
It exposes `ObjectNode` to public surface

Reply:
all of the new public constructors are in the implementation package not in the public surface area.